README.md

```diff
@@ -103,6 +103,8 @@ All list responses are fully paginated.
 
 Setting `downloadFiles: true` in the plugin configuration enables downloading of files associated with File objects, and images on Sku and Product objects. A Gatsby [File node](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-source-filesystem#how-to-query) is created for each downloaded file, and references are placed on the `localFiles` field of their Stripe nodes.
 
+When downloading these files this plugin will by default use the same authorization header used in the HTTP request for fetching your stripe details. This can cause some issues with platforms used to host public files (e.g. Google Cloud Platform Storage Buckets) where you will receive a 401 Unauthorized since this header won't match anything the platform knows about. As such you can choose to remove the auth header using the setting `auth: false`. Currently you can not set custom auth headers due to limitations in [gatsby-source-filesystem](https://www.gatsbyjs.org/packages/gatsby-source-filesystem/), however this limitation may be removed in the future.
```

Comment:
In the "How to use" section of the README, there's an example config. I know that `auth: false` is an optional element to add to the config, but putting it in the example would be useful, I think. Then, after this line `Enable downloading files associated with your Stripe data by setting downloadFiles to true.`, you could add a sentence like: 'Also, you can add `auth: false` as shown below to... [insert brief explanation about why, linking to the 'Download Files' section again]'. Let me know if that makes sense!